io/driver/driver_parameter.py.data/driver_parameter_slate.yaml

```diff
@@ -0,0 +1,17 @@
+# $ modinfo -p be2net
+#   num_vfs:Number of PCI VFs to initialize (uint)
+#   rx_frag_size:Size of a fragment that holds rcvd data. (ushort)
+
+module: be2net
+interface: eth1
+host_ip:
+netmask:
+peer_ip:
+sysfs_check_required: 0
+module_param_name: "rx_frag_size"
+value: !mux
+    4096:
```

Comment:
@vaishnavibhat Will `rx_frag_size_4096` be better for readability in `avocado run` output ?

Reply:
@vaishnavibhat please paste job.log and `avocado run` output also please